main.pm

```diff
@@ -29,7 +29,7 @@ sub load_install_tests() {
     # for now when testing from git only tests the webui itself, not worker
     # interaction
     return 1 if check_var('OPENQA_FROM_GIT', 1);
-    loadtest "install/openqa_worker.pm";
+    loadtest "install/openqa_worker.pm" unless check_var('OPENQA_FROM_BOOTSTRAP', 1);
```

Comment:
```suggestion
    loadtest "install/openqa_worker.pm" unless get_var('OPENQA_FROM_BOOTSTRAP');
```

I created https://github.com/os-autoinst/os-autoinst-distri-openQA/pull/118 to fix the other cases of boolean equality check that should be avoided